qcodes/dataset/sqlite/queries.py

```diff
@@ -25,7 +25,7 @@
 from qcodes.dataset.descriptions.versioning import serialization as serial
 from qcodes.dataset.descriptions.versioning import v0
 from qcodes.dataset.descriptions.versioning.converters import new_to_old, old_to_new
-from qcodes.dataset.guids import generate_guid, parse_guid
```

Comment:
Do we plan to deprecate the generate_guid function or is there still a need to keep it?

Reply:
Not at the moment this is still used to generate guids when a new dataset is created